features/step_definitions/metrics_steps.rb

```diff
@@ -0,0 +1,3 @@
+Given /^assuming (\d+) days is set to retrieve the reports by day$/ do |nb_days|
```

Comment:
That doesn't mean anything. Replace with "assuming reports by day metrics are calculated for X days by default".